src/Factories/FunFunFactory.php

```diff
@@ -81,7 +81,7 @@
 use WMDE\Fundraising\Frontend\Infrastructure\Repositories\LoggingSubscriptionRepository;
 use WMDE\Fundraising\Frontend\Infrastructure\TemplateBasedMailer;
 use WMDE\Fundraising\Frontend\Infrastructure\TokenGenerator;
-use WMDE\Fundraising\Frontend\Infrastructure\TwigCachePurger;
+use WMDE\Fundraising\Frontend\Infrastructure\AllOfTheCachePurger;
```

Comment:
Yeah, I also can't think of something good that is not open to misinterpretation